integration/yarn/src/test/java/alluxio/yarn/ApplicationMasterTest.java

```diff
@@ -109,6 +110,7 @@
       ImmutableMap.<String, String>builder()
           .put("CLASSPATH", ENV_CLASSPATH)
           .put("ALLUXIO_HOME", ApplicationConstants.Environment.PWD.$())
+          .put("ALLUXIO_RAM_FOLDER", ApplicationConstants.Environment.LOCAL_DIRS.$())
```

Comment:
Is the idea here to mount the ramdisk inside the Yarn container so that Yarn cleans it up on exit?